Tests/OptimizelyTests-Common/OdpManagerTests.swift

```diff
@@ -81,6 +81,36 @@ class OdpManagerTests: XCTestCase {
         XCTAssertNil(manager.segmentManager)
     }
     
+    @available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
+    func testConfigurations_disableOdp_aync_await() async throws {
+        let manager = OdpManager(sdkKey: sdkKey,
+                                 disable: true,
+                                 cacheSize: cacheSize,
```

Comment:
We can remove these tests when async functions in OdpManager removed.

Reply:
done